docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,283 @@
+# Using configtx.yaml to create a channel genesis block
+
+A channel is created by building a channel genesis block that contains the initial configuration of the channel. The **channel configuration** is stored on the ledger, and governs all the subsequent blocks that are added to the channel. The channel configuration specifies which organizations are channel members, the ordering nodes that can create new blocks, and the set of organizations that need to agree to channel updates. The channel configuration also specifies which users have permission to access the data on the channel ledger, and which organizations need to approve a chaincode definition.
```

Comment:
As I read through this I found myself continuously wondering if this is referring to the genesis block for the system channel or the application channel.  The answer is both! Down in the profiles section, line 133 you address this. I feel like a sentence at the beginning here to give some context would be helpful..

Reply:
Changed the last sentence to this one: "A Fabric network is formed when a group of organizations create the genesis block of the system channel that defines the ordering service of the network and the blockchain [consortium](../glossary.html#consortium). Peer organizations then can then use the system channel to create the genesis blocks of application channels. The permission to access the network, or update a channel to change the membership or governance, is encoded in the genesis blocks of the system channel and the application channels of the network."